internal/concurrent/concurrent.go

```diff
@@ -36,6 +36,15 @@ func NewGroup(ctx context.Context) *Group {
 	}
 }
 
+// Go runs func in backround.
+// Func should return when Group.Context is cancelled/done.
+func (g *Group) Go(f func()) {
+	g.Group.Go(func() error {
```

Comment:
sorry, it's fine for this PR but add a TODO to review this. the whole struct makes no sense anymore. An errgroup is only really useful if you want to stop a whole group of goroutines when the first one exits, otherwise you can just use a simple `WaitGroup`

Reply:
Let me change this right now (switch to just WaitGroup). I still want to keep this `concurrent.Group` struct 'cause it implements `bps.Subscription` (well, Closer - to tell goroutines to exit).

Reply:
Will do in next PR